src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,7 +127,39 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
-
+        
+        
+        JButton button = new JButton(Constant.messages.getString("fuzz.httpfuzzer.results.toolbar.button.export"));
+        button.setIcon(new ImageIcon(HttpFuzzResultsContentPanel.class.getResource("/resource/icon/16/115.png")));
+        button.addActionListener((new AbstractAction() {
+        private static final long serialVersionUID = 1L;
+
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				WritableFileChooser chooser = new WritableFileChooser();
+				chooser.setSelectedFile(new File(Constant.messages.getString("fuzz.httpfuzzer.results.toolbar.button.export.defaultName")));
+				if (chooser.showSaveDialog(View.getSingleton().getMainFrame()) == WritableFileChooser.APPROVE_OPTION) {
+					String file = chooser.getSelectedFile().toString();
+					if (!file.endsWith(".csv")) {
+						file += ".csv";
+					}
+					try (CSVPrinter pw = new CSVPrinter(new BufferedWriter(new FileWriter(chooser.getSelectedFile())), CSVFormat.DEFAULT)) {

```

Comment:
Ideally it should use UTF-8, instead of relying on default charset.
(Files.newBufferedWriter(...) allows to specify the charset ;)